### ISSUE TITLE
password

### ISSUE THREAD

**Issue by batmanpolsl** [Benjamin-Dobell/nvidia-update#45]
sorry but I am really new in this. I have tried to run the script but I can't complete it. It asking for the password - what is it?

**Comment by itsnoa04** [comment]
yes, this is your sudo user password. What was the password you gave to your user when you initially signed up?

Note: it also might be the same as your login password.
